### DISCUSSION TITLE
How to hide speed control buttons in video player overlay

### DISCUSSION THREAD

**Discussion by ghost** [qnblackcat/uYouPlus#1261] [Q&A]
![5b3f4cf47d76fdedbf610fa79e7bcb81](https://github.com/qnblackcat/uYouPlus/assets/67296948/0c3f4535-0cc0-4909-b4e3-69347fa79db0)

**Comment by KalamxBD** [comment]
yep here is the solution: https://imgur.com/a/PqeA58M
